object/object_copy.go

```diff
@@ -0,0 +1,180 @@
+package object
+
+import (
+	"github.com/nspcc-dev/neofs-api-go/v2/object"
+	"github.com/nspcc-dev/neofs-api-go/v2/refs"
+	v2session "github.com/nspcc-dev/neofs-api-go/v2/session"
+)
+
+func copyByteSlice(sl []byte) []byte {
```

Comment:
you copy slice of _something_ more that once, how about  😎**generic**😎 `copySlice`?

Reply:
I'd leave it for 1.21.

Reply:
with 1.21 it is even better and (if my suggestion is applied) could be fixed with `sed`.
not critical for me